src/library_workerfs.js

```diff
@@ -91,7 +91,7 @@ addToLibrary({
       },
       setattr(node, attr) {
         for (const key of ["mode", "atime", "mtime", "ctime"]) {
-          if (attr[key]) {
+          if (typeof attr[key] === 'number') {
```

Comment:
How about `!= 'undefined'` here and elsewhere ?

Reply:
typeof null is object though.

Reply:
Ah but we could do `attr[key] != null` because `0 != null` is `true` and `undefined != null` is `false`.